Documentation/FrameworkOverview.md

```diff
@@ -8,65 +8,124 @@ learning about new modules and finding more specific documentation.
+interrupt/cancel the work associated Signal.
 
-### Subscription
+### Observers
 
-A **subscriber** is anything that is waiting or capable of waiting for events
-from a [signal](#signals). Within RAC, a subscriber is represented as any object
-that conforms to the [RACSubscriber][] protocol.
+An **Observer** is anything that is waiting or capable of waiting for events
+from a [signal](#signals). Within RAC, an observer is represented as an instance
+of the [SinkOf][] struct with an input type of Event.
 
-A **subscription** is created through any call to
-[-subscribeNext:error:completed:][RACSignal], or one of the corresponding
-convenience methods. Technically, most [RACStream][] and
-[RACSignal][RACSignal+Operations] operators create subscriptions as well, but
+<!-- TODO: Since this is a high level description, we have avoided to write about
+the generic type signatures for Signal / SignalProducer / SinkOf (Observer), but 
+the sentence above feels awkeward. Maybe make it pseudo-concrete by writing
+`SinkOf<Event>` ? -->
+
+A signal can be observed by calling its `observe` method, providing either a
+sink or callbacks for the different types of events as a parameter.
+
+<!-- TODO: As I understand from the source code, this next statement is still true.
+Is it relevant enough to be here? -->
+Technically, most [Signal][] and
+[SignalProducer][] operators create subscriptions as well, but
 these intermediate subscriptions are usually an implementation detail.
 
-Subscriptions [retain their signals][Memory Management], and are automatically
-disposed of when the signal completes or errors. Subscriptions can also be
-[disposed of manually](#disposables).
+### Action
+
+An **Action**, represented by the [Action][] class, will do some work when
+executed with an _Input_. During or after execution, zero or more _Output_
+values are generated. Alternatively, an _Error_ may be generated.
+
+Actions are suited to perform side-effecting work as the user interacts with
+the app.
+
+<!-- TODO: How to call >thing< ? It was 'action' before, but "some action triggers the Action" is strange.
+'event' would fit, but could be mistaken for the Events associated with a signal -->
+Usually the thing triggering an Action is UI-driven, like when a button is
+clicked. Actions can also be automatically disabled based on a signal, and this
+disabled state can be represented in a UI by disabling any controls associated
+with the action.
+
+For interaction with UIKit or AppKit GUI controls such as `NSControl` or 
+`UIControl`, RAC provides [CocoaAction][] to wrap Action with KVO or 
+Cocoa Bindings.
+
+### Property
+
+A **Property**, represented by the [PropertyType][Property] protocol, allows
+observation of its changes.
+
+<!-- TODO: Refer to KVO and say this replaces KVO? Or is this to specific / irrelevant? -->
 
+The current value of a property can be obtained from the `value` getter. The
+`producer` getter returns a [SignalProducer](#signal producers) that will send
```

Comment:
I _think_ you need to replace spaces with dashes for header anchors - `[SignalProducer](#signal-producers)`